### PR TITLE
Screenshot added in Readme

### LINKED ISSUE #4: Add screenshots to Readme
Use the test exports to add screenshots of graphs and text outputs.

### PR DESCRIPTION
**Pull request by asaraf7**
Closes #4

### CODE REVIEW

**Comment by JRavi2** [comment]
@asaraf7 Thanks for the PR!

Since we don't require the images for any other purpose, you can drag and drop the images from any file manager while editing the readme file. Github will create a link for you which will be in the format of: 
`https://user-images.githubusercontent.com/******/********.file_format`

Also, the images are a bit too large, could you reduce their sizes a bit?

**Comment by asaraf7** [comment]
Your recommendations implemented

File: README.md [hunk 1]
```diff
@@ -33,3 +35,14 @@ To know about the available options run the command:
 To test changes, enter the `tests` directory and run the following command:
 
     pytest tests.py
+    
+    
+## Output
```
Comment (JRavi2) [inline README.md]: Can we "Output" to something like "Example plots"? What do you think?

File: README.md [hunk 2]
```diff
@@ -33,3 +35,14 @@ To know about the available options run the command:
...
+
+The program can provide various graphs upon request (wherever possible)
+For example:
+![img2](https://user-images.githubusercontent.com/84586244/145166287-5643a572-ac17-4795-bdc6-43ab82a6274c.jpeg)
+![img3](https://user-images.githubusercontent.com/84586244/145166324-194aad63-d901-41fe-bb28-9f00f465ce71.jpeg)
+![img4](https://user-images.githubusercontent.com/84586244/145166146-c8186b67-8f9b-4572-8f20-329fdb3bcbbc.jpeg)
```
Comment (JRavi2) [inline README.md]: I think we could make this look a bit better by centering the images a bit.
A suggestion: 
```md
<p align="center"> 
<img src="https://user-images.githubusercontent.com/84586244/145166287-5643a572-ac17-4795-bdc6-43ab82a6274c.jpeg" />
<img src="https://user-images.githubusercontent.com/84586244/145166324-194aad63-d901-41fe-bb28-9f00f465ce71.jpeg" />
<img src="https://user-images.githubusercontent.com/84586244/145166146-c8186b67-8f9b-4572-8f20-329fdb3bcbbc.jpeg" />
</p>
```

File: README.md [hunk 3]
```diff
@@ -33,3 +35,14 @@ To know about the available options run the command:
...
 
     pytest tests.py
+    
+    
+## Output
+
+The program can provide various graphs upon request (wherever possible)
+For example:
+![img2](https://user-images.githubusercontent.com/84586244/145166287-5643a572-ac17-4795-bdc6-43ab82a6274c.jpeg)
+![img3](https://user-images.githubusercontent.com/84586244/145166324-194aad63-d901-41fe-bb28-9f00f465ce71.jpeg)
```
Comment (JRavi2) [inline README.md]: This image has a thin black line on the left edge. Could you crop this out?

**Review by JRavi2 — CHANGES_REQUESTED** [review]
This is looking great! Just a few small changes before merging.

File: README.md (hunk 1 above)
Comment (asaraf7) [reply README.md]: Sure. What the program will give as output is already mentioned in the description

**Review by JRavi2 — APPROVED** [review]

**Comment by JRavi2** [comment]
Thanks, looks good. Merging!

**Comment by JRavi2** [comment]
Also, congrats on getting your first PR merged :tada:

**Comment by asaraf7** [comment]
> Also, congrats on getting your first PR merged 🎉

Thank you. It means a lot!

**Pull request merged**